doc/Install.md

```diff
@@ -135,6 +135,10 @@ For more information during the installation, you can use this command line opti
 
     bin/console autoinstall -v
 
+### Prepare .htaccess file
+
+Copy .htaccess-dist to .htaccess (be careful under stupid M$ Windows) to have working mod-rewrite again. If you have installed Friendica into a sub directory, like /friendica/ set this path in RewriteBase accordingly.
```

Comment:
I understand the feeling but the official Friendica documentation should be as neutral as possible.